src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -9056,7 +9056,7 @@ void CalcHeatBalanceInsideSurf2CTFOnly(EnergyPlusData &state,
 void sumSurfQdotRadHVAC(EnergyPlusData &state)
 {
     for (int surfNum : state.dataSurface->allGetsRadiantHeatSurfaceList) {
-        auto &thisSurfQRadFromHVAC = state.dataHeatBalFanSys->surfQRadFromHVAC(surfNum);
+        auto const &thisSurfQRadFromHVAC = state.dataHeatBalFanSys->surfQRadFromHVAC(surfNum);
         state.dataHeatBalSurf->SurfQdotRadHVACInPerArea(surfNum) = thisSurfQRadFromHVAC.HTRadSys + thisSurfQRadFromHVAC.HWBaseboard +
                                                                    thisSurfQRadFromHVAC.SteamBaseboard + thisSurfQRadFromHVAC.ElecBaseboard +
                                                                    thisSurfQRadFromHVAC.CoolingPanel;
```

Comment:
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:8470]:(style),[constVariable],Variable 'Surface' can be declared as reference to const
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:8655]:(style),[constVariable],Variable 'thisSpace' can be declared as reference to const
[src/EnergyPlus/HeatBalanceSurfaceManager.cc:9059]:(style),[constVariable],Variable 'thisSurfQRadFromHVAC' can be declared as reference to const